src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -428,7 +428,7 @@ export class TerminalService implements ITerminalService {
 		// windows
 		this.terminalInstances.forEach(instance => instance.dispose(!isWindows));
 
-		this._localTerminalService!.setTerminalLayoutInfo(undefined);
```

Comment:
I don't think we want to throw if `this._localTerminalService` is undefined

Reply:
This looks good